replace_address_test.py

```diff
@@ -35,7 +35,7 @@ class BaseReplaceAddressTest(Tester):
 
     def _setup(self, n=3, opts=None, enable_byteman=False, mixed_versions=False):
         debug("Starting cluster with {} nodes.".format(n))
-        self.cluster.populate(n)
+        self.cluster.populate(n, use_vnodes=not DISABLE_VNODES)
```

Comment:
Is this change necessary? It's not clear to me that vnodes would have an effect on configurable ports

Reply:
I think I was fixing some other annoying bug. Should have added a comment. I'll test without it and report back.